### ISSUE TITLE
Rewrite manpage with mdoc(7)

### ISSUE THREAD

**Issue by Artoria2e5** [rrthomas/mmv#6]
mdoc is a different, not quite new, macro package for writ ing manual pages. Its main point is to write in a more semantic way that serves search systems and HTML output better.

Among other things, mdoc(7) has you:
* use macros instead of manual indents in code samplea
* use specialized macros for referring to other man pages (turns into hyperlink in PDF/HTML)

(The table should be rewritten with tbl(7) anyways, man or mdoc.)

**Comment by rrthomas** [comment]
I would be happy to get a pull request for that.

**Comment by xtaran** [comment]
Speaking with my `mmv` Debian package maintainer hat on: Not happy. `mdoc` seems neither very common nor is it packaged for Debian. :-/

Additionally [I can't find that system with searching for "mdoc" in the internet](https://duckduckgo.com/?q=%22mdoc%22&atb=v11&ia=web).

**Comment by rrthomas** [comment]
@xtaran, I thought there are plenty of mdoc pages in Debian already, and it's already included in groff.

**Comment by xtaran** [comment]
Ah, ok, I thought it was a separate package. Also I did not find a single file in Debian with `mdoc` als file name base. I now see that the actual file name for the man page is `groff_mdoc` which is not matched by my search `\bmdoc\b' as underscore counts as word-character.

**Comment by rrthomas** [comment]
Yes, I think this is a remnant of the BSD/System V split which was swept under the carpet many years ago!

**Comment by xtaran** [comment]
Anyway, if it's part of `groff`, there's no objection anymore from my side. :-)
